temporalio/client.py

```diff
@@ -3400,6 +3461,16 @@ def __init__(
                 if pair.key.name in self.untyped_search_attributes:
                     # We know this is mutable here
                     del self.untyped_search_attributes[pair.key.name]  # type: ignore
+            self.static_summary = (
+                raw_info.user_metadata.summary
```

Comment:
Isn't this a payload? I think you're going to have to change this up a bit to move all of the raw info conversion stuff into `_from_proto` and make that method `async def`. May also need a test that confirms schedule describe returns the same workflow summary/details it was created with.

Reply:
This is working exactly the same way memo does when reading from raw info, for example. The tests that compare schedule before/after seem to confirm it works properly but I'll double check

Reply:
Ah, now I see `Optional[Union[str, temporalio.api.common.v1.Payload]]`, you're good